### ISSUE TITLE
feat: enable passing MarkerClusterGroup options

### ISSUE THREAD

**Issue by ManUtopiK** [nuxt-modules/leaflet#136]
[Here](https://github.com/nuxt-modules/leaflet/blob/03ab5b77c816704671b1bfa81cf0226561d89234/src/runtime/composables/useLMarkerCluster.ts#L28) we should be able to pass the [`markerClusterGroup` options](https://github.com/Leaflet/Leaflet.markercluster/tree/master?tab=readme-ov-file#options).
Something like :
```js
...

interface Props {
  leafletObject: Map
  markers: MarkerProps[]
  options: MarkerClusterGroupOptions
}

export const useLMarkerCluster = async (props: Props) => {
  // Get Leaflet from the window object
  const L = window.L

  // Lazy-load leaflet.markercluster
  // Importing it at the top level will cause errors because it could be loaded before the Leaflet library
  const { MarkerClusterGroup } = await import('leaflet.markercluster')

  // Initialize marker cluster
  const markerCluster = new MarkerClusterGroup(props.options)
```

**Comment by Gugustinette** [comment]
Oh that's true ! I'll work on that as soon as possible 👍

**Comment by Gugustinette** [comment]
Released in [v1.2.5](https://github.com/nuxt-modules/leaflet/releases/tag/1.2.5) 👍

**Comment by ManUtopiK** [comment]
Hi! Thanks for your works on it!
The definition of the interface `MarkerClusterGroupOptions` is missing many options. All options seem visible here : https://github.com/Leaflet/Leaflet.markercluster/blob/c0f055bd5dcd1d6a733090d0cf024d7362d77bc8/src/MarkerClusterGroup.js#L7

Sadly, leaflet.markercluster isn't typed. But you can find all types for the option here : https://github.com/DefinitelyTyped/DefinitelyTyped/blob/61576c3c5bcb6518f86224fcb0a2314ac68e4ebb/types/leaflet.markercluster/index.d.ts#L36

Instead of re-creating markercluster group options definition, maybe it will make more sense to use [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped). It will be possible to import types definition for leaflet and leaflet.markercluster. Ex:
```bash
pnpm add -D @types/leaflet
pnpm add -D @types/leaflet.markercluster
```
So `MarkerClusterGroupOptions` will be available in leaflet type :
`import type { MarkerOptions, Map, Marker, MarkerClusterGroupOptions } from 'leaflet'`

**Comment by Gugustinette** [comment]
This is merged and released in [v1.2.6](https://github.com/nuxt-modules/leaflet/releases/tag/1.2.6), thanks @ManUtopiK !
